docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
       # From observation, although we don’t have complete confidence in the root cause,
       # it appears that localstack sometimes misses API requests when run in parallel.
       TF_CLI_ARGS_apply: "--parallelism=1"
+      TERRAFORM_VERSION: ${TERRAFORM_VERSION:-latest}
```

Comment:
☝️ I believe this is necessary so certain acceptance tests can be conditionally skipped based on the Terraform version in use, based on `TERRAFORM_VERSION`.

Reply:
Why not use SupportsStateReplaceProvider()? It directly represents the skip condition of the test.

Reply:
OK. The test for state replace-provider requires special setups, and we want to check the version before initialization 👌